dask/bag/text.py

```diff
@@ -63,9 +68,16 @@ def read_text(
 
     >>> b = read_text('largefile.txt', blocksize='10MB')  # doctest: +SKIP
 
+    Get file paths of the bag by setting include_path=True
+
+    >>> b = read_text('myfiles.*.txt', include_path=True) # doctest: +SKIP
+    >>> b.pluck(1).take(1) # doctest: +SKIP
+    ('.../myfiles.0.txt',)
```

Comment:
I think it'd be a bit easier to understand with something like

```
>>> b.pluck(1)
(('first line of the first file\n', 'myfiles.0.txt'),)
```

Why is the `...` present in your output? Are the paths relative or absolute?

Reply:
Think it should be
```
>>> b.take(1)
(('first line of the first file\n', '.../myfiles.0.txt'),)
```
Paths are absolute. The `...` is just ellipsis for path. Open to suggestions for alternatives to `...`

Reply:
Maybe something generic like `/home/dask/myfiles.0.txt`?